### ISSUE TITLE
[Bug Report]  App crashes after opening

### ISSUE THREAD

**Issue by SimonIT** [SapuSeven/BetterUntis#33]
**Description**
App crashed immediately after opening

<details>
  <summary><b>Screenshots</b></summary>

  
![Screenshot_20191222-013539](https://user-images.githubusercontent.com/11720038/71315379-c5f4cc80-245b-11ea-8944-d930deaacf3d.png)

</details>

**Additional information**
 - Android 8.1.0
 - BetterUntis 3.2.0

**Comment by SapuSeven** [comment]
Deobfuscated stack trace:
```
java.util.NoSuchElementException: 
  at kotlin.collections.ArraysKt___ArraysJvmKt.c (ArraysKt___ArraysJvmKt.java:21)
  at com.sapuseven.untis.receivers.LessonEventSetupKt.withLast (LessonEventSetupKt.java:2)
  at com.sapuseven.untis.receivers.AutoMuteSetup.onLoadingSuccess (AutoMuteSetup.java:2)
  at com.sapuseven.untis.receivers.LessonEventSetup$loadTimetable$$inlined$let$lambda$1.a (LessonEventSetup.java:11)
  at com.sapuseven.untis.helpers.timetable.TimetableLoader.loadFromCache (TimetableLoader.java:26)
  at com.sapuseven.untis.helpers.timetable.TimetableLoader.access$loadFromCache (TimetableLoader.java:26)
  at com.sapuseven.untis.helpers.timetable.TimetableLoader$load$1.invokeSuspend (TimetableLoader.java:26)
  at kotlin.coroutines.jvm.internal.BaseContinuationImpl.a (BaseContinuationImpl.java:9)
  at kotlinx.coroutines.DispatchedTask.run (DispatchedTask.java:11)
```

**Comment by SapuSeven** [comment]
Fixed in version [3.2.1](https://github.com/SapuSeven/BetterUntis/releases/tag/v3.2.1).
Google Play Beta update will be available in a few hours.
